pkgs/misc/vscode-extensions/python/default.nix

```diff
@@ -37,19 +38,26 @@ let
       sha256 = languageServerSha256;
     };
   };
+
+  libcMusl = runCommand "libc-musl-x86_64" {} ''
+    mkdir -p $out/lib
+    cp -pdv ${musl}/lib/libc.so $out/lib/libc.musl-x86_64.so.1
+    ln -s $out/lib/libc.musl-x86_64.so.1 $out/lib/libc.musl-x86_64.so
```

Comment:
Should this go into the `musl` derivation?

Reply:
most likely, but that can be a separate PR

Reply:
https://github.com/NixOS/nixpkgs/pull/89112